src/server/common/upload-status.js

```diff
@@ -0,0 +1,21 @@
+const UploadStatus = {
+  Initiated: 'initiated',
+  Quarantined: 'quarantined',
+  Scanned: 'scanned',
+  Delivered: 'delivered',
+  Acknowledged: 'acknowledged'
+}
+
+function canBeUploaded(status) {
+  return !status || status === UploadStatus.Initiated
+}
+
+function canBeMoved(safe, status) {
```

Comment:
This feels like it should return a Boolean? Maybe wrap the return in `Boolean()` - at the moment if `safe` or `status` are `null` or `undefined` it will return that

Reply:
Will do. Rather will try